moveit_simple/src/moveit_simple.cpp

```diff
@@ -219,16 +217,115 @@ bool Robot::getJointSolution(const Eigen::Affine3d &pose, double timeout,
   return getIK(pose, local_seed, joint_point, timeout);
 }
 
+bool Robot::getJointSolution(const Eigen::Affine3d &pose, const std::string& custom_tool_frame,
+                             double timeout, const std::vector<double> & seed,
+                             std::vector<double> & joint_point) const
+{
+
+  // Ideally "tool0" should be replaced with joint_group_->getEndEffectorName()
+  std::string moveit_tool_link = "tool0";
```

Comment:
This needs to be fixed before this is accepted.

Reply:
I'll make sure to fix this issue when I send out the PR for overloading addTrajPoint(...) and the execute(...) functions.